CHANGELOG.md

```diff
@@ -1,60 +1,60 @@
 <a name="Pending Release"></a>
-## [Pending Release](https://github.com/lightstep/lightstep-tracer-java/compare/master...0.12.5)
+## [Pending Release](https://github.com/lightstep/lightstep-com.lightstep.tracer-java/compare/master...0.12.5)
```

Comment:
oh I just meant the link is linking to `lightstep-tracer-java` without the `common` at the end